openapi/components/parameters/cityCouncilDistrictIdQueryParam.yaml

```diff
@@ -0,0 +1,9 @@
+name: cityCouncilDistrictId
+required: false
+in: query
+schema:
+  type: string
+  pattern: ^([0-9]{1,2})$
+  example: '25'
+  nullable: true
```

Comment:
Same as below

Reply:
This conflicts with Tim's request [above](https://github.com/NYCPlanning/ae-zoning-api/pull/411#discussion_r1941383063).

Changing this changes the `FindCapitalProjectsQueryParams` type to disallow the option for `cityCouncilDistrictId` from being `null`, which then prevents us from using `null` to represent the lack of `cityCouncilDistrictId`.